ui/apps/platform/src/Containers/Policies/Table/PoliciesTable.tsx

```diff
@@ -373,62 +392,76 @@ function PoliciesTable({
+                            labelAndNotifierIdsForTypes,
+                            notifierIds
+                        );
+                        const exportPolicyAction: ActionItem = {
+                            title: 'Export policy to JSON',
+                            onClick: () => exportPoliciesHandler([id]),
+                        };
+                        const actionItems = hasWriteAccessForPolicy
+                            ? [
+                                  {
+                                      title: 'Edit policy',
+                                      onClick: () => onEditPolicy(id),
+                                  },
+                                  {
+                                      title: 'Clone policy',
+                                      onClick: () => onClonePolicy(id),
+                                  },
+                                  disabled
+                                      ? {
+                                            title: 'Enable policy',
+                                            onClick: () => enablePoliciesHandler([id]),
+                                        }
+                                      : {
+                                            title: 'Disable policy',
+                                            onClick: () => disablePoliciesHandler([id]),
+                                        },
+                                  exportPolicyAction,
+                                  {
+                                      isSeparator: true,
+                                  },
+                                  {
+                                      title: isDefault
+                                          ? 'Cannot delete a default policy'
+                                          : 'Delete policy',
+                                      onClick: () => setDeletingIds([id]),
+                                      isDisabled: isDefault,
+                                  },
+                              ]
+                            : [exportPolicyAction];
+                        const rowIndex = rowIdToIndex[id];
+                        return (
+                            <Tbody
+                                key={id}
+                                style={{
+                                    borderBottom: '1px solid var(--pf-c-table--BorderColor)',
+                                }}
+                                isExpanded={isExpanded}
+                            >
+                                <Tr>
+                                    <Td
+                                        expand={{
+                                            rowIndex,
+                                            isExpanded,
+                                            onToggle: () => expandedRowSet.toggle(id),
+                                        }}
```

Comment:
I took the first pictures, with the new column and their sortings, before I implemented the expanded Description, because I was planning to PR those two pieces separately. Then, I realized that the expanding was straightforward to add and easily fit in one set of changes.